docs/changelog-fragments.d/README.md

```diff
@@ -1,50 +1,43 @@
 <!-- markdownlint-disable first-line-heading -->
 
-(_ansible_language_server_adding_changelog_fragments)=
+(ansible_language_server_adding_changelog_fragments)=
 
 ## Adding change notes with your PRs
 
-It is very important to maintain a log for news of how
```

Comment:
This huge rewrite contributes to the complexity of synchronizing this text across projects 👎

Reply:
@ansible/devtools team never agreed on something like this. We do not have a reference sync source and the reality is that even if we would do, we could implement the same formatting on the other one.

Basically, i see the comment as diverging from the scope of the PR and no real reasons for excluding this file, especially as it will render any attempt to touch it with vscode a nightmare as prettier will reformat it.